src/coreclr/interpreter/compiler.cpp

```diff
@@ -807,6 +853,29 @@ void InterpCompiler::EmitCode()
     m_compHnd->setBoundaries(m_methodInfo->ftn, m_ILToNativeMapSize, m_pILToNativeMap);
 }
 
+void InterpCompiler::BuildGCInfo(InterpMethod *pInterpMethod)
+{
+#ifdef FEATURE_INTERPRETER
```

Comment:
Doesn't really make sense to build the interpreter library if `FEATURE_INTERPRETER` is disabled. Currently this flag is defined as `$<IF:$<CONFIG:Debug,Checked>,1,0>` making it awkward to disable the interpreter library build (I think because this condition is resolved later during build). I think we should have interpreter built also in Release mode anyway. @janvorli What are your thoughts ?